changelog/2801.bugfix.rst

```diff
@@ -0,0 +1 @@
+Helioviewer client now uses Helioviewer API version 2 exclusively.
```

Comment:
I think the file needs to be renamed. At this point we are far past bug fixes. `2801.breaking.rst` would be my suggestion

Reply:
Yeah, true. We are too far from just a bug fix. I will change it